Site Settings: Highlight support docs on Empty Site page

jeremeylduvall opened the pull request:
Currently, we have [an empty site support page](https://support.wordpress.com/empty-site/) that instructs users how to delete posts and pages. However, on the Empty Site page for a given site, those instructions aren't very apparent. This PR adds a button to highlight those instructions.

To test, checkout this branch and visit http://calypso.localhost:3000/settings/start-over/site.wordpress.com for your given site.

Previously, we just had a "Contact Support" link:

![screen shot 2016-08-31 at 12 50 34 pm](https://cloud.githubusercontent.com/assets/7240478/18142165/8d6d70aa-6f79-11e6-8f09-a27009aaae86.png)

The updated version will have a primary Empty Site link to our support doc.

![screen shot 2016-08-31 at 12 51 11 pm](https://cloud.githubusercontent.com/assets/7240478/18142176/9c8597ac-6f79-11e6-9e2a-94dd900b0f98.png)

Meant to address #7594.

Test live: https://calypso.live/?branch=fix/7594-start-over-button

jordwest commented:
Tested this out and the code looks good.

At first glance it looked as if clicking the button would actually empty the site, I'd be a bit afraid of clicking it without seeing that it links to a support doc. I've added a label for design review, maybe some others can weigh in on this?

mtias commented:
Agreed with @jordwest. Perhaps "How to empty site" as the label? It should probably not be primary either, and we should add the external icon link since the doc is not in calypso.

folletto commented:
> At first glance it looked as if clicking the button would actually empty the site, I'd be a bit afraid of clicking it without seeing that it links to a support doc.

+1, the wording implies an active action by the system.

```
[ How to empty site ] [ Contact support ]
```

This looks good to me, however, in the full page context, this might work even better:

```
[ Follow the steps ] [ Contact support ]
```

I'd suggest this option because the text just above says "Alternatively, you can delete all content from your site by following the steps here." so the action should resonate. 

In the presence of that button, and with a clear resonance, I'd remove the link from the text too.

jeremeylduvall commented:
Appreciate the feedback. The initial verbiage was a bit confusing now that I think about it. What about this rendition? I removed the primary button, added the external icon, and changed the button text to "Follow the Steps" (removed the link from the paragraph text as well):

![screen shot 2016-09-08 at 8 46 19 pm](https://cloud.githubusercontent.com/assets/7240478/18373943/d2282bd0-7605-11e6-9cb3-c9582b49068a.png)

folletto commented:
Design wise looks good to me 👍

jeremeylduvall commented:
Yep—mostly just a label change and then a class switch to `.is-external`. Provided I didn't completely botch the Sass formatting, I think this is good to 🚢 .

mtias commented on client/my-sites/site-settings/action-panel/style.scss:
```diff
@@ -76,3 +76,16 @@ a.settings-action-panel__body-text-link {
+		margin-right: 8px;
+
+		&:after {
+			@include noticon('\f442');
```
We don't use noticons anymore.

mtias commented:
@jeremeylduvall external links should be constructed with https://wpcalypso.wordpress.com/devdocs/design/external-link or with a gridicon inside the button.

jeremeylduvall commented:
Updated to use a Gridicon inside the button. Current appearance:

<img width="739" alt="screen shot 2016-09-17 at 10 18 29 am" src="https://cloud.githubusercontent.com/assets/7240478/18609830/1c96940c-7cc0-11e6-96f2-87fbbdbc834b.png">

jeremeylduvall commented:
I would love to remove the external linking, but moving the steps into Calypso is part of a larger discussion of moving our Support docs into Calypso as well. I don't think that's on the immediate radar. It could be possible (the pages would need to be available logged out and logged in), but until we make a definitive decision, it feels like moving these steps into Calypso would just create two reference points to update if one of the steps changes.
